arch/x86/kvm/pkvm/vmx/host_vmx.c

```diff
@@ -128,14 +185,25 @@ void pkvm_host_vmexit_main(struct vcpu_vmx *vmx)
 		handle_xsetbv(vcpu);
 		skip_instruction = true;
 		break;
+	case EXIT_REASON_INTERRUPT_WINDOW:
+		handle_irq_window(vcpu);
+		break;
+	case EXIT_REASON_PREEMPTION_TIMER:
+		pin_controls_clearbit(vmx, PIN_BASED_VMX_PREEMPTION_TIMER);
```

Comment:
nit: Do this in a separate function handle_preemption_timer() (just for consistency with other vmexits)?